apps/meteor/tests/e2e/saml.spec.ts

```diff
@@ -313,6 +332,18 @@ test.describe('SAML', () => {
 		});
 	});
 
+	test('Redirect after login', async ({ page }) => {
```

Comment:
Can we add 2 more test cases, please?

- One to test the old behavior is preserved, redirect to home when no `redirectUrl` is present.
- Another one with a not valid URL (some URL that does not start the site_Url)

Reply:
I added the first test case you suggested for the old behavior and it's working fine. However, I can't think of a way to test it with an invalid URL, since that'd require us to access RC login from an external URL (that's possible locally with ngrok, for example, but not sure if there's a way to implement an e2e test for this)